source/_posts/2020-03-14-2020-using-vyper-with-embark.md

```diff
@@ -0,0 +1,204 @@
+title: Using Vyper with Embark
+summary: "Vyper has gained some traction on Solidity as an Ethereum smart contract language, especially with support going in to ETH2. Learn how to use Embark to use Vyper smart contracts in a &ETH;App."
+author: eric_mastro
+categories:
+  - Report
+layout: blog-post
+image: '/assets/images/vyper_simple_storage_with_logo.png'
+---
+
+![Vyper Language](/assets/images/vyper_simple_storage_with_logo.png)
+
+## Why Vyper?
+[Vyper](https://github.com/vyperlang/vyper) is a contract-based approach to writing Ethereum smart contracts, and is aimed at bringing ["increased security and reability through a minimalist approach"](https://ethereumdevelop.com/ethereum-solidity-vyper/) to the smart contract landscape.
```

Comment:
I don't think you need the comma after "Ethereum smart contracts", but I might be wrong.